scripts/scripts.js

```diff
@@ -127,6 +128,7 @@ function loadDelayed() {
 }
 
 async function loadPage() {
+  loadTargetOffers('sitesinternal');
```

Comment:
Moving this up, just below the import theoretically a) saves a few nanoseconds of JS execution, and b) also keeps the logic together so it's easier to see at a glance

Reply:
> also keeps the logic together so it's easier to see at a glance

I put it here for exactly this reason. I wouldn't expect to see any important logic among the import statements.